crates/ra_assists/src/assists/auto_import.rs

```diff
@@ -64,12 +64,16 @@ pub(crate) fn auto_import<F: ImportsLocator>(
         return None;
     }
 
-    ctx.add_assist_group(AssistId("auto_import"), "auto import", || {
-        proposed_imports
-            .into_iter()
-            .map(|import| import_to_action(import, &position, &path_to_import_syntax))
-            .collect()
-    })
+    ctx.add_assist_group(
+        AssistId("auto_import"),
+        format!("Import {}", path_to_import.syntax()),
```

Comment:
> path_to_import.syntax()

NIT: I think you could have used the `path_to_import_syntax` variable here.

Reply:
Yeah, I just noticed that. Thanks.